2022-04-08 08:08:31,959 INFO    MainThread:34514 [internal.py:wandb_internal():92] W&B internal server running at pid: 34514, started at: 2022-04-08 08:08:31.959534
2022-04-08 08:08:31,962 INFO    WriterThread:34514 [datastore.py:open_for_write():77] open: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/run-2_tbl_1_lr_grand_citeseer.wandb
2022-04-08 08:08:31,966 DEBUG   SenderThread:34514 [sender.py:send():235] send: header
2022-04-08 08:08:31,966 DEBUG   SenderThread:34514 [sender.py:send():235] send: run
2022-04-08 08:08:32,276 DEBUG   HandlerThread:34514 [handler.py:handle_request():141] handle_request: check_version
2022-04-08 08:08:32,278 INFO    SenderThread:34514 [dir_watcher.py:__init__():169] watching files in: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files
2022-04-08 08:08:32,278 INFO    SenderThread:34514 [sender.py:_start_run_threads():812] run started: 2_tbl_1_lr_grand_citeseer with start time 1649405311
2022-04-08 08:08:32,278 DEBUG   SenderThread:34514 [sender.py:send():235] send: summary
2022-04-08 08:08:32,279 INFO    SenderThread:34514 [sender.py:_save_file():947] saving file wandb-summary.json with policy end
2022-04-08 08:08:32,279 DEBUG   SenderThread:34514 [sender.py:send_request():249] send_request: check_version
2022-04-08 08:08:32,582 DEBUG   HandlerThread:34514 [handler.py:handle_request():141] handle_request: run_start
2022-04-08 08:08:32,589 DEBUG   HandlerThread:34514 [meta.py:__init__():37] meta init
2022-04-08 08:08:32,589 DEBUG   HandlerThread:34514 [meta.py:__init__():51] meta init done
2022-04-08 08:08:32,590 DEBUG   HandlerThread:34514 [meta.py:probe():211] probe
2022-04-08 08:08:32,596 DEBUG   HandlerThread:34514 [meta.py:_setup_git():201] setup git
2022-04-08 08:08:32,614 DEBUG   HandlerThread:34514 [meta.py:_setup_git():208] setup git done
2022-04-08 08:08:32,614 DEBUG   HandlerThread:34514 [meta.py:_save_pip():55] save pip
2022-04-08 08:08:32,615 DEBUG   HandlerThread:34514 [meta.py:_save_pip():69] save pip done
2022-04-08 08:08:32,615 DEBUG   HandlerThread:34514 [meta.py:_save_conda():76] save conda
2022-04-08 08:08:33,283 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_created():217] file/dir created: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/wandb-summary.json
2022-04-08 08:08:33,284 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_created():217] file/dir created: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/conda-environment.yaml
2022-04-08 08:08:33,284 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_created():217] file/dir created: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/requirements.txt
2022-04-08 08:08:34,475 DEBUG   HandlerThread:34514 [meta.py:_save_conda():86] save conda done
2022-04-08 08:08:34,475 DEBUG   HandlerThread:34514 [meta.py:probe():249] probe done
2022-04-08 08:08:34,482 DEBUG   SenderThread:34514 [sender.py:send():235] send: files
2022-04-08 08:08:34,482 INFO    SenderThread:34514 [sender.py:_save_file():947] saving file wandb-metadata.json with policy now
2022-04-08 08:08:34,490 DEBUG   HandlerThread:34514 [handler.py:handle_request():141] handle_request: stop_status
2022-04-08 08:08:34,490 DEBUG   SenderThread:34514 [sender.py:send_request():249] send_request: stop_status
2022-04-08 08:08:34,777 DEBUG   SenderThread:34514 [sender.py:send():235] send: telemetry
2022-04-08 08:08:34,777 DEBUG   SenderThread:34514 [sender.py:send():235] send: alert
2022-04-08 08:08:35,227 INFO    Thread-11 :34514 [upload_job.py:push():137] Uploaded file /tmp/tmpi0y6rxzewandb/ruevdbvq-wandb-metadata.json
2022-04-08 08:08:35,284 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/conda-environment.yaml
2022-04-08 08:08:35,284 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_created():217] file/dir created: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/output.log
2022-04-08 08:08:35,285 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_created():217] file/dir created: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/wandb-metadata.json
2022-04-08 08:08:37,285 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/output.log
2022-04-08 08:08:41,287 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/output.log
2022-04-08 08:08:43,288 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/output.log
2022-04-08 08:08:45,289 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/output.log
2022-04-08 08:08:47,290 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/output.log
2022-04-08 08:08:49,290 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/output.log
2022-04-08 08:08:49,779 DEBUG   HandlerThread:34514 [handler.py:handle_request():141] handle_request: stop_status
2022-04-08 08:08:49,779 DEBUG   SenderThread:34514 [sender.py:send_request():249] send_request: stop_status
2022-04-08 08:08:51,291 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/output.log
2022-04-08 08:08:53,292 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/output.log
2022-04-08 08:08:55,293 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/output.log
2022-04-08 08:08:57,294 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/output.log
2022-04-08 08:08:59,295 INFO    Thread-7  :34514 [dir_watcher.py:_on_file_modified():230] file/dir modified: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/files/output.log
2022-04-08 08:08:59,377 WARNING MainThread:34514 [internal.py:wandb_internal():155] Internal process interrupt: 1
2022-04-08 08:09:00,442 WARNING MainThread:34514 [internal.py:wandb_internal():155] Internal process interrupt: 2
2022-04-08 08:09:00,443 ERROR   MainThread:34514 [internal.py:wandb_internal():158] Internal process interrupted.
2022-04-08 08:09:00,446 INFO    HandlerThread:34514 [handler.py:finish():790] shutting down handler
2022-04-08 08:09:00,446 DEBUG   SenderThread:34514 [sender.py:send():235] send: telemetry
2022-04-08 08:09:00,446 INFO    WriterThread:34514 [datastore.py:close():281] close: /home/administrator/hieu/graph-neural-pde/src/wandb/run-20220408_080831-2_tbl_1_lr_grand_citeseer/run-2_tbl_1_lr_grand_citeseer.wandb
2022-04-08 08:09:00,447 INFO    SenderThread:34514 [sender.py:finish():1107] shutting down sender
2022-04-08 08:09:00,447 INFO    SenderThread:34514 [dir_watcher.py:finish():283] shutting down directory watcher
2022-04-08 08:09:00,737 INFO    MainThread:34514 [internal.py:handle_exit():82] Internal process exited
